### PR TITLE
rec: Prepare for recursor 4.3.0-beta2 release

### PR DESCRIPTION
**Pull request by omoerbeek**
### Checklist

I have:
- [X] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [ ] compiled this code
- [ ] tested this code
- [ ] included documentation (including possible behaviour changes)
- [ ] documented the code
- [ ] added or modified regression test(s)
- [ ] added or modified unit test(s)

### CODE REVIEW

**Comment by omoerbeek** [comment]
Draft blog post: https://powerdnsblog.wordpress.com/?p=2988&preview=true

File: pdns/recursordist/docs/changelog/4.3.rst [hunk 1]
```diff
@@ -1,6 +1,44 @@
...
+    :tickets: 8642
+
+    Fix wrong zoneCuts caused by cache only lookup.
+
+  .. change::
+    :tags: Enhancement
+    :pullreq: 8675
+    :tickets: 8646
+
+    Increase default maxqperq.
```
Comment (rgacogne) [inline pdns/recursordist/docs/changelog/4.3.rst]:
```suggestion
    Increase default max-qperq.
```

**Comment by rgacogne** [comment]
Blog post looks good, except that the link for opening an issue skips our template. I suggest replacing it with https://github.com/PowerDNS/pdns/issues/new/choose instead.

**Comment by omoerbeek** [comment]
Comments processed for changelog and blog post.

**Pull request merged**